python/semantic_kernel/connectors/ai/open_ai/services/open_ai_chat_completion_base.py

```diff
@@ -64,6 +67,7 @@ def get_prompt_execution_settings_class(self) -> type["PromptExecutionSettings"]
         return OpenAIChatPromptExecutionSettings
 
     @override
+    @trace_chat_completion(MODEL_PROVIDER_NAME)
```

Comment:
Do we have a concept sample that shows/explains how to configure telemetry?

Reply:
Yes. Coming out today, or tomorrow at the latest.